es/index.js

```diff
@@ -102,7 +108,7 @@ export default function axiosBunyan(axios, {
 }) {
   const logRequest = (config) => {
     // eslint-disable-next-line no-param-reassign
-    config.timestamp = new Date().toISOString();
+    config[namespace] = Object.assign({ timestamp: Date.now() }, config[namespace]);
```

Comment:
It also made more sense to store the timestamp itself and not the ISO representation for the logs.